engine/src/main/java/org/entando/entando/web/component/ComponentController.java

```diff
@@ -53,5 +55,17 @@ public ResponseEntity<SimpleRestResponse<List<ComponentUsageDetails>>> extractCo
         List<ComponentUsageDetails> details = this.service.extractComponentUsageDetails(components);
         return new ResponseEntity<>(new SimpleRestResponse<>(details), HttpStatus.OK);
     }
-
+    
+    @RestAccessControl(permission = Permission.SUPERUSER)
+    @DeleteMapping(value = "/allInternals/delete", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
a delete http verbs with delete inside the url is not the best way to do a REST service (in the url you should put resource not action)